### PR TITLE
Modal Styles added to be used in search and login

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--nedbank--hlxsites.hlx.live/content/nedbank/za/en/personal/home
- After: https://overlay-class--nedbank--hlxsites.hlx.live/content/nedbank/za/en/personal/home

### CODE REVIEW

File: styles/styles.less [hunk 1]
```diff
@@ -96,6 +96,19 @@ body {
...
+		overflow: hidden;
+	}
+
+	.modal {
+        width: 100%;
+        height: 100%;
+        z-index: 1050;
+        top: 64px;
+        position: fixed;
+        display: flex;
```
Comment (sdmcraft) [inline styles/styles.less:110]: Should this be kept `hidden` by default and show up only on click?

Comment (jindaliiita) [reply styles/styles.less:110]: I have applied display: none to corresponding less file and then applying this class on click.
Similarly for search also we can set by default display: none in corresponding less file and then on click can use this class.

Comment (sdmcraft) [reply styles/styles.less:110]: > I have applied display: none to corresponding less file and then applying this class on click.

You  mean there's another file (not included in this PR) where its set to `none`. Maybe include them in the PR as well to get the complete picture.

Comment (jindaliiita) [reply styles/styles.less:110]: Yes but i think that should be part of other PR which will contain work corresponding to the login section.
I have only set this in login.less file
.login-section {
    display: none;
}

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**